script.js

```diff
@@ -141,32 +141,43 @@ function HideModal() { // eslint-disable-line no-unused-vars
 }
 
 // #region
-const form = document.getElementsByClassName('contact-form')[0];
-const emailField = document.getElementById('email');
-const setError = (message) => {
-  const errorContainer = form.querySelector('.error');
-  errorContainer.innerHTML = message;
-  errorContainer.classList.add('error');
-};
+// Get the form element from the doc
+const form = document.querySelector('form');
+
+// Add an "submit" event listener to the form element
+form.addEventListener('submit', (event) => {
+  // Prevent the default form submission behavior
```

Comment:
kindly remember to fulfill the below project requirement 
> After filling the form, and not submitting it, the entered values should be saved in the browser. 

here it means as I fill out the form the data should be saved in the local storage immediately without any other action such as reloading, or submitting,...... and consider  setting the whole input value, not just the first letter to the local storage check the image below 
![za](https://user-images.githubusercontent.com/60146030/224441410-9f04a60d-47cb-4f8f-92cc-adfbf73f2bde.PNG)
kindly consider implementing that it will make your project be on a high standard as it will meet all the requirements